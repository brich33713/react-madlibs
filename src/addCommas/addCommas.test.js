import { fireEvent, render, getByText, getByTestId, getByLabelText } from '@testing-library/react';
import AddCommas from './addCommas'

it('renders without crashing',()=>{
    render(<AddCommas />)
})

it("matches snapshot",()=>{
    const {asFragment} = render(<AddCommas/>)
    expect(asFragment()).toMatchSnapshot()
})