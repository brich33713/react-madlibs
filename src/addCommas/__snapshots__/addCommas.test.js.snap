// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Add Commas
    </h1>
    <form>
      <label
        for="input"
      >
        Enter Number:
      </label>
      <input
        data-testid="input"
        type="text"
      />
      <button
        data-testid="convert"
      >
        Convert Number
      </button>
    </form>
    <p
      data-testid="output"
    />
  </div>
</DocumentFragment>
`;
